Proposed charter amendments (#342)
web-platform-tests/interop: charter.md

The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established.

> The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus at a minimum once per calendar year. There's no term limit; the same individual can be reappointed as chair. If the team is unable to meet due to absence of a chair, the Core Team may appoint one directly.

Reviewer: ```suggestion
The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus at a minimum once per calendar year. There's no term limit; the same individual can be reappointed as chair. If the team is unable to meet due to absence of a chair, the [Core Team](https://github.com/orgs/web-platform-tests/teams/wpt-core-team) may appoint one directly.
```

This link only works for those in the core team (and admins and org owners) but it's better than nothing I think.

Reviewer: I've filed https://github.com/web-platform-tests/wpt/issues/40212, we can use a better link later if we document the core team.

Author: I don't think we should use a link that will point to an error page for the majority of people.

The team is responsible for maintaining [Interop 2021](https://wpt.fyi/interop-2021), [2022](https://wpt.fyi/interop-2022), [2023](https://wpt.fyi/interop-2023), and defining future interoperability efforts and metrics.

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Reviewer: ```suggestion
If the Team is unable to reach consensus, and therefore unable to progress in these areas, the Team's functions revert to the core team, and this charter is annulled.
```

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Author: But what counts as "progress"? If the team is succeeding in maintaining the existing metrics, but failing at defining future interoperability efforts and metrics, is that still making "progress"?

Like, I think your argument is that it isn't, but I don't think that's clear.

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Reviewer: Ping @jgraham

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Reviewer 2: I think the answer is "I don't know, and therefore I don't want the charter to take a strong position on this". I think in practice any situation where the team was unable to reach agreement for an extended period of time would cause alarm bells to ring, and change the situation in a way that would either cause the core team to intervene or cause participants to leave the project in a way that would undermine its credibility in the wider community. 

In particular the restriction to "publish new metrics" wouldn't make sense in the hypothetical situation where we replaced annual metrics with a continuously updated "living metric". It's not clear to me if such a situation would be precluded by your proposed wording (would it be possible to reach consensus to _not_ publish new metrics?) or whether, if it was not, the proposed wording would cover a situation where we couldn't agree on how to _update_ an existing metric (vs publish a new one).

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Reviewer: Could everyone live with either saying nothing, or merely noting that the WPT core team has the power to dissolve the team / annul the charter? It sounds quite ominous, and is true even if we don't write it down, so my weak preference would be to say nothing.

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Author: I think saying nothing is fine to me?

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Reviewer 2: Now that you've brought it up, I think some statement of the fact that the interop group (process-wise) ultimately answers to the wpt core team would be useful. That could be something on the wpt side that explains how sub-groups work.

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Author: In general terms, I think @jgraham is right that we should put that in some Core Team document; prior to having such a thing, I'm happy to add a statement of fact to the Interop charter.

@jgraham, do you have any specific suggested text here?

> If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.

Author: Having discussed this in the meeting, sounds like we have rough consensus to say nothing here. Going to resolve this thread, then.
